geniza/corpus/models.py

```diff
@@ -442,6 +443,9 @@ def all_tags(self):
 
     all_tags.short_description = "tags"
 
+    def alphabetized_tags(self):
+        return self.tags.order_by(Lower("name"))
+
```

Comment:
I feel like this would make more sense to put on the tag manager, but I'm not sure it's worth the extra effort to extend and customize at this point.

There is some possibly related work with taggit that I should look into — turning on `TAGGIT_CASE_INSENSITIVE` or possibly even forcing tags to lower case (which would simplify this!). The particular issue I'm thinking of is #257 ; we should probably discuss whether that should be a technical or data entry change.